.wordlist.txt

```diff
@@ -334,6 +334,8 @@ ympd
 ZeroPi
 zr
 de facto
```

Comment:
I also try&fail learned here, neither white spaces not dashes work in the word list, it's always two single words that need to be added then, hence the entry `de facto` above can be removed, the below two are required.

Reply:
agree, I will do this